tests/analytics/test_forecasting_v2_compression_breakout.py

```
@@ -1126,6 +1126,21 @@ def test_instrument_tick_size_negative_raises():
             instrument=make_instrument(tick_size=-0.1)))
 
 
+@pytest.mark.parametrize("tick_a,tick_b", [(0.10, 0.50), (1.0, 2.5)])
```

Ruff (0.16.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)